### PR TITLE
Create Executive vote - June 13, 2024.md

### PR DESCRIPTION
**Pull request by 0xldr**
To Do:

- [x] add spark address and etherscan link
- [x] obtain ecosystem approval for relevant entries and add links where appropriate

### CODE REVIEW

File: governance/votes/Executive vote - June 13, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - June 13, 2024.md:44]: I think it would be easier to read this sentence changing the order of its elements to: "If this executive proposal passes, deposits on the old Starknet Dai Bridge will be disabled as part of the transition to a new bridge version maintained by Starkgate".

File: governance/votes/Executive vote - June 13, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - June 13, 2024.md:96]: I'd change the verb to the noun (by adding "of" after "Transfer") to better fit the flow of the sentence. This comment applies also to the second bullet.

File: governance/votes/Executive vote - June 13, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - June 13, 2024.md:108]: Capital letter for "Mainnet"

File: governance/votes/Executive vote - June 13, 2024.md
Comment (MakerJanSky) [reply governance/votes/Executive vote - June 13, 2024.md:44]: I think it is fine in either format.

File: governance/votes/Executive vote - June 13, 2024.md
Comment (MakerJanSky) [reply governance/votes/Executive vote - June 13, 2024.md:96]: This is the standard language that has been used on executives and polls. We can consider changing in the style guide, but I do not think it is necessary here.

File: governance/votes/Executive vote - June 13, 2024.md
Comment (MakerJanSky) [reply governance/votes/Executive vote - June 13, 2024.md:108]: In the past this has only been capitalised in titles. Please consider for style guide inclusion, but it should not block approval of this copy.

**Review by MakerJanSky — COMMENTED** [review]
Approving once Spark Proxy Spell details are added. Nice work, ldr.

**Review by MakerJanSky — APPROVED** [review]
Nice work, ldr.

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:79]: We have an extra column here

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:127]:
```suggestion
* Supply Cap (`max`): **50,000 weETH**
```

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:128]:
```suggestion
* Supply Cap (`gap`): **5,000 weETH**
```

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:129]:
```suggestion
* Supply Cap (`ttl`): **12 hours**
```

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:131]:
```suggestion
* Borrow Cap (`max`): N/A
```

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:132]:
```suggestion
* Borrow Cap (`gap`): N/A
```

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - June 13, 2024.md:133]:
```suggestion
* Borrow Cap (`ttl`): N/A
```

**Review by blimpa — CHANGES_REQUESTED** [review]
LGTM

**Review by VoteWizard — CHANGES_REQUESTED** [review]
Non-blocking suggestions

**Comment by VoteWizard** [comment]
# Spell Coordination Sheet Review

## Checklist

- [x] Are all fields filled in?
- [x] Do contents match the discussion from the GovOps Meeting?
    - [x] Confirm all items were discussed at GovOps.
    - [x] Confirm no items from the discussion were omitted.
- [x] Are incomplete items (awaiting final figures) marked with a clear `TODO`?
- [x] If there is a SubDAO Proxy Spell:
    - [x] Is the proxy spell address listed in a "Trigger $SubDAO Proxy Spell at" line?
    - [x] Are SubDAO Spell contents seperated ([example](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=684547605&range=J26))?
- [x] Are the DAI transfers rounded to the nearest DAI (no decimal points)?
- [x] Are MKR transfers listed to exactly two decimal places?
- [x] Is the Checksum for DAI and MKR accurate?
- [x] Are Spell Items listed in order of importance for an average MKR voter?
 - Note: It appears that the last item was added later, and the Drafter chose not to alter the order of the items previously confirmed by the content liaisons
- [x] Have all items been approved publicly by the appropriate party?
 - Note: The Governance point decided that it was not necessary to require approval from Ecosystem for the "USDP Jar Housekeeping" instruction since it was a minor housekeeping action and the same code that was previously approved to perform the same housekeeping action in previous executive votes was going to be used


---



## Content
- [x] Check the title refers to the most impactful/important actions included in the proposal.
- [x] Check that the summary refers to all actions included in the proposal.

- [x] Check that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Check that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the reviewers of the executive code are listed in the preamble (at the entity level).

- [x] Check that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Check that it is clearly stated which items are affected by the GSM Delay.
- [x] Check that the GSM Delay period is communicated.
- [x] Check that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Check each item is present in the proposal details section
- [x] Check items are ordered by judged impact/importance
 - Note: It appears that the last item was added later, and the Drafter chose not to alter the order of the items previously confirmed by the content liaisons

- [x] For each item
	- [x] Check the executive summary contains a high-level overview of the item.
	- [x] Check that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Check that the copy contains a link to the reasoning for its inclusion.
	- [x] Check that the copy contains a link to the authorization for its inclusion.
	- [x] Check that the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Check that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Check that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Check that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Check that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Final Checks

- [x] Read the executive copy in full, with the above checklists and general readability in mind. Raise any issues with the author.

File: governance/votes/Executive vote - June 13, 2024.md
Comment (0xldr) [reply governance/votes/Executive vote - June 13, 2024.md:79]: we don't see an extra column

**Review by VoteWizard — APPROVED** [review]
I approve this PR

File: governance/votes/Executive vote - June 13, 2024.md
Comment (blimpa) [reply governance/votes/Executive vote - June 13, 2024.md:79]: The table should have three columns, but this line defines four:

> |1-----------------|2-----------------------------------------------------------------------------------------------------------------------|3--------------------|4--------------|

I'm pointing this out because I don't know how the engine will render it

**Pull request merged**
